client/src/components/Invitation.js

```diff
@@ -45,8 +71,18 @@ export default function FormDialog() {
 
   /*push emails to emailList*/
   const addEmail = () => {
-    if (emailList.findIndex((email) => email == emails) < 0)
-      setEmailList([...emailList, emails]);
+    // if (emailList.findIndex((email) => email == email) < 0)
+    if (!emailList.includes(email)) {
+      setEmailList([...emailList, email]);
+      setEmail("");
+    } else return;
```

Comment:
If we don't add the email we probably want to give an error message, maybe clear the input